backend/src/types/models/complaint-updates/complaint-updates.ts

```diff
@@ -0,0 +1,15 @@
+import { Point } from "typeorm";
+
+export class ComplaintUpdatesDto {
```

Comment:
for your properties unless you know there's going to be a null value you may want to use the `?` operator on the type instead.  This will help with when we run into silly null problems, plus it will kick vscode to remind you the property you're using could be undefined.